subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/internal/finalizer/LegacyFinalizer.kt

```diff
@@ -22,43 +22,38 @@ internal class LegacyFinalizer(
 
     override fun finalize(testsExecutionResults: TestsScheduler.Result) {
 
-        val reportedTestsResult = report.getTests(testsExecutionResults.testSuite.testsToRun.map { it.test.name })
+        val testResults = report.getTests(
+            initialSuiteFilter = testsExecutionResults.testSuite.testsToRun.map { it.test.name }
+        ).getOrElse { emptyList() }
```

Comment:
But know we will green when we have failed tests?
Or we will fail when read inmemory report

Reply:
it is the same logic as it was before; we will fail on LostTestDeterminer in that case